packages/gasket-data/lib/index.js

```diff
@@ -0,0 +1,8 @@
+const deepFreeze = require('deep-freeze');
+
+let content = (document.getElementById('GasketData') || {}).textContent;
+if (content) {
+  content = deepFreeze(JSON.parse(content));
```

Comment:
What benefits do we gain by freezing this data? I'm personally all for minimizing data mutation, but this could lead to unexpected runtime errors, not to mention it will add additional runtime overhead while traversing the data and running `Object.freeze`. Maybe this is too prescriptive.

Reply:
Just trying to minimize abuse and set expectations that this should be read-only, used merely for transferring data from server to client.

It shouldn't be needed in production builds anyway. Maybe it's something we can conditionally set during development if we see it being abused.

I can leave it out for now.